### ISSUE TITLE
Dashboard "Open in Visual Studio Code" uses Windows Filesystem instead of WSL

### ISSUE THREAD

**Issue by msanchezdev** [docker/for-win#6472]
I'm not very sure about where I should open the issue, if here or vscode repo
  - [x] I have tried with the latest version of my channel (Stable or Edge)
  - [ ] I have uploaded Diagnostics
  - Diagnostics ID:

### Expected behavior
Visual Studio Code needs to open with project folder inside WSL2.

### Actual behavior
Visual Studio Code opens with the original path as if it were on the Windows Filesystem.

### Information
  - Windows Version: Windows 10 Pro 2004 (Build 19041.208)
  - Docker Desktop Version: 2.3.0.0 (44472) Edge Channel
  - Are you running inside a virtualized Windows e.g. on a cloud server or on a mac VM: Windows PC with WSL2 distro.

#### docker-compose.yaml
```
version: '3'

services:
  server:
    image: httpd:2.4
    volumes:
      - ./public:/usr/local/apache2/htdocs
    ports:
      - 8080:80
```

1. start docker-compose project
2. hit *Open in Visual Studio Code* in the dashboard

**Comment by simonferquel** [comment]
Thanks for reporting the issue! I'll put that on our backlog, it might be very tricky to fix though.
In the meantime, I am affraid you need to open vscode manually by entering your wsl2 distro and hitting `code .` from the correct directory.

**Comment by msanchezdev** [comment]
Not sure how it saves compose information but I guess it can check for  the path delimiter or something like that.

**Comment by mat007** [comment]
The fix has now just been released in Docker Desktop Stable 2.3.0.3, I’ll go ahead and close this issue.
